src/js/jsx/sections/libraries/LibraryBar.jsx

```diff
@@ -37,10 +37,25 @@ define(function (require, exports, module) {
     var LibraryBar = React.createClass({
         mixins: [FluxMixin],
 
-        createNewElement: function () {
+        addImageAsset: function () {
             this.getFlux().actions.libraries.createElementFromSelectedLayer();
         },
 
+        addCharacterStyle: function () {
```

Comment:
These should probably be private methods. Their JSDoc annotations should have `@private`.